cgsmiles/read_fragments.py

```diff
@@ -102,7 +110,18 @@ def fragment_iter(fragment_str, all_atom=True):
             mol_graph.add_node(0, element="H", bonding=bonding_descrpt[0])
             nx.set_node_attributes(mol_graph, bonding_descrpt, 'bonding')
         elif all_atom:
-            mol_graph = pysmiles.read_smiles(smile)
+            try:
+                mol_graph = pysmiles.read_smiles(smile)
```

Comment:
```suggestion
                mol_graph = pysmiles.read_smiles(smile, reinterpret_aromatic=False)
```

Reply:
Unfortunately, not. The error is raised already at the parsing stage of the cgsmiles string. You categorically do not allow non-cyclic aromatics. We could modify pysmiles though to allow this if 'reinterpret_aromatic' is set to False